src/Features/Core/Portable/Diagnostics/Analyzers/ProjectDiagnosticAnalyzer.cs

```diff
@@ -19,5 +21,13 @@ internal abstract class ProjectDiagnosticAnalyzer : DiagnosticAnalyzer
         public sealed override void Initialize(AnalysisContext context)
         {
         }
+
+        /// <summary>
+        /// This lets vsix installed <see cref="DocumentDiagnosticAnalyzer"/> or <see cref="ProjectDiagnosticAnalyzer"/> to
+        /// specify priority of the analyzer. Regular <see cref="DiagnosticAnalyzer"/> always comes before those 2 different types.
+        /// Priority is ascending order and this only works on HostDiagnosticAnalyzer meaning Vsix installed analyzers in VS.
+        /// This is to support partner teams (such as typescript and F#) who want to order their analyzer's execution order.
+        /// </summary>
+        public virtual int Priority { get; } = DefaultPriority;
```

Comment:
:bulb: The second backing field here isn't needed. You can use `=>` instead of `{ get; } =`.

Reply:
done